setup.cfg

```diff
@@ -40,17 +40,11 @@ install_requires =
 	waveorder==1.0.0rc0
 	pycromanager==0.19.2
 	click>=8.0.1
-	pyyaml>=5.4.1
-	tqdm>=4.61.1
```

Comment:
The zarr converter uses this.

Reply:
And it's pretty light, so I don't mind keeping it

Reply:
napari already depends on this and we depend on napari, e.g. we do not depend on NumPy for the same reason

Reply:
Gotcha! Thanks.